tests/fixtures/apps/demultiplexing/flowcell_runs/201203_A00689_0200_AHVKJCDRXX/HVKJCDRXX_demultiplex.stderr

```diff
@@ -0,0 +1,13 @@
+[20210403115107] On node: compute-0-2.local
```

Comment:
do we need this fixture? or can live without it on public repo?

Reply:
this is parsed in the code (perhaps not this exact line) so I would like to keep it, don't really see the problem.